apps/db/lib/db/dataset.ex

```diff
@@ -364,19 +398,19 @@ defmodule DB.Dataset do
     |> Repo.all()
   end
 
-    def fetch_history_metadata(bucket, obj_key) do
-      bucket
-        |> S3.head_object(obj_key)
-        |> ExAws.request!
-        |> Map.get(:headers)
-        |> Map.new(fn {k, v} -> {String.replace(k, "x-amz-meta-", ""), v} end)
-        |> Map.take(["format", "title", "start", "end", "updated-at", "content-hash"])
-    end
+  def fetch_history_metadata(bucket, obj_key) do
+    bucket
+      |> S3.head_object(obj_key)
+      |> ExAws.request!
+      |> Map.get(:headers)
+      |> Map.new(fn {k, v} -> {String.replace(k, "x-amz-meta-", ""), v} end)
+      |> Map.take(["format", "title", "start", "end", "updated-at", "content-hash"])
+  end
 
   ## Private functions
   @cellar_host ".cellar-c2.services.clever-cloud.com/"
 
-  defp history_resource_path(bucket, name), do: Path.join(["http://", bucket <> @cellar_host, name])
+  defp history_resource_path(bucket, name), do: Path.join(["http://", bucket, @cellar_host, name])
```

Comment:
ah nop, I think changing this will add an additional `/` that makes the url invalid